hosting/single/nginx.conf

```diff
@@ -0,0 +1,116 @@
+user                 www www;
+error_log            /etc/nginx/logs/error.log;
+pid                  /etc/nginx/logs/nginx.pid;
+worker_processes     auto;
+worker_rlimit_nofile 8192;
+
+events {
+  worker_connections  1024;
+}
+
+http {
```

Comment:
Can we look into having this run through the prod hbs nginx file? That'll give us a win for maintenance so that we don't and can't accidentally forget changes to this file. This looks pretty much like the prod config file ran through the processor so it should be achievable enough

Reply:
Just that having _yet another_ nginx config floating about doesn't sound like a good idea

Reply:
Yeah I was hoping to do that at some point, just ran out of time on lab day and didn't want to start spreading the work out across multiple days - was going to try and merge all the configs into one when taking on the full issue itself! Could be a bit of a fiddly task, needs a good bit of testing since it has potential to break all self hosting/cloud deployments.

Reply:
Sounds good! be great to get them all merged